nodejs/README.md

```diff
@@ -1,214 +1,114 @@
+**Suggestions** contain examples that have worked well on a project or two;
+they're not widely used enough to be defaults, but are worth considering.
 
 
-## Twelve-factor web applications
+## Versions
 
-[12factor.net](https://12factor.net/) describes a twelve-factor methodology for
-building scalable web applications. By following the methodology, your
-application implements a generic contract with the platform. This contract
-allows it to be scaled and managed by the platform without intimate knowledge of
-how the application works.
+Node.js supports certain versions as Long-term Support (LTS) versions. We have
+**standardized** to using the [current active
+LTS](https://github.com/nodejs/LTS#lts-schedule1) version, currently v8.x.
 
-[cloud.gov](https://cloud.gov/) is the preferred platform at 18F. Building your
-Node.js application with the Twelve-factor methodology in mind will ensure your
-application is designed for cloud.gov.
 
-
-### Best practices
-
-#### Recommend
-
-- Build your application on top of cloud.gov.
-- Read the [Twelve-factor methodology](https://12factor.net/) and keep it in
-  mind as you build your application.
-
-
-## Choosing a version
-
-### Node.js
-
-Node.js supports certain versions as Long-term Support (LTS) versions. We
-recommend the [current active LTS](https://github.com/nodejs/LTS#lts-schedule1)
-version, currently v6.x.
-
-
-### ECMAScript
-
-ES5, ES6, ES7? JavaScript is an evolving language and is defined by
-[ECMAScript](https://en.wikipedia.org/wiki/ECMAScript). Modern browsers support
-ES5, but when writing code on the server, you don't have to write
+JavaScript is an evolving language defined by
+[ECMAScript](https://en.wikipedia.org/wiki/ECMAScript). Modern browsers
+support ES5, but when writing code on the server, you don't have to write
 browser-compatible code. With `--harmony`, Node.js [supports many ES6 and ES7
-features](http://node.green/).
-
-You can also use the same build tools that you use on the front end, like
-webpack and browserify with babel, to transpile your code. This allows you to
-fill in the feature gaps that are not yet supported by Node.js. This also means
-your browser code and server code use the same ECMAScript features.
-
-
-### Best practices
-
-
-#### Recommend
-
-- Use the [current active LTS](https://github.com/nodejs/LTS#lts-schedule1)
-  version of Node.js, currently v6.x.
-
-
-#### Consider
-
-- What syntax are you using on the front end? Consider using the same syntax and
-  features for both the  front end and back end code.
-- Many developers write JavaScript in the browser (ES5) even if they aren't
-  a JavaScript developer. Using a syntax that many folks are familiar with
-  allows more folks to jump in.
-- Front end build tools make it possible to choose a syntax you want and
-  polyfill the missing features in the browser.
-
-
-## Frameworks
+features](http://node.green/). Similarly, on both the front end _and_ backend,
```

Comment:
We use "front end" but not "back end"?

Reply:
Ah, good point. I tend to not use a space, but I know the ~frontend~ front end folks consistently use one. I'll add one for consistency.